### DISCUSSION TITLE
Github repo link is outdated

### DISCUSSION THREAD

**Discussion by Tristan971** [corazawaf/coraza#276] [General]
Jfyi the repo has [www.coraza.io](https://www.coraza.io/) as main link (right sidebar) which is currently NXDOMAIN 😄

![2022-07-04 12_34_33-corazawaf_coraza_ OWASP Coraza WAF is a golang modsecurity compatible web applic](https://user-images.githubusercontent.com/875533/177146680-4ef18252-aaa7-40c6-90c0-033af68e03bb.png)

probably want to update that

**Comment by jptosso** [comment]
Hey, thank you for your report. The domain was migrated yesterday and I forgot to copy the www CNAME entry.

Fixed now
